package.json

```diff
@@ -24,6 +25,29 @@
   },
   "homepage": "https://github.com/colossal-digital/acssemble#readme",
   "dependencies": {
-    "emotion": "^10.0.14"
+    "emotion": "^10.0.14",
+    "rollup-plugin-commonjs": "^10.0.1",
```

Comment:
This should be a `devDependency`, as should the remainder of the `rollup-plugin-*`.